packages/flutter_tools/test/general.shard/web/devfs_web_test.dart

```diff
@@ -392,6 +397,48 @@ void main() {
     expect((await response.read().toList()).first, utf8.encode('main() {}'));
   }));
 
+  test('serves flutter_bootstrap.js without useLocalCanvasKit', () => testbed.run(() async {
+    globals.fs.file(globals.fs.path.join(
+      globals.artifacts!.getHostArtifact(HostArtifact.flutterJsDirectory).path,
+      'flutter.js',
+    ))..createSync(recursive: true)..writeAsStringSync('flutter.js content');
+
+    final Response response = await webAssetServer.handleRequest(
+      Request('GET', Uri.parse('http://foobar/flutter_bootstrap.js'))
+    );
+
+    expect(response.statusCode, 200);
+    final String body = await response.readAsString();
+    expect(body, isNot(contains('"useLocalCanvasKit":true')));
```

Comment:
Should this be broader, like `isNot(contains('useLocalCanvasKit'))`? What if someone changes the code and now it contains `"useLocalCanvasKit":false`? Or `"useLocalCanvasKit":shouldUseLocalCanvasKit`?